src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/external/ResettlementPassportDeliusApiService.kt

```diff
@@ -23,25 +23,19 @@ import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.data.deliusapi.
 import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.data.deliusapi.Manager
 import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.data.deliusapi.MappaDetail
 import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.data.deliusapi.PersonalDetail
-import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.jpa.repository.PrisonerRepository
 import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.service.convertNameToTitleCase
 import java.time.LocalDate
 import java.time.OffsetDateTime
 
 @Service
 class ResettlementPassportDeliusApiService(
   private val rpDeliusWebClientCredentials: WebClient,
-  private val prisonerRepository: PrisonerRepository,
 ) {
 
   companion object {
     private val log = LoggerFactory.getLogger(this::class.java)
   }
 
-  fun findCrn(nomsId: String): String? {
-    return prisonerRepository.findByNomsId(nomsId)?.crn
-  }
-
   @Cacheable("resettlement-passport-delius-api-get-crn", unless = "#result == null")
```

Comment:
Maybe we should change this to cache on null? Otherwise we might end up calling this API a lot for the cases where there is no CRN available?

Reply:
Removed